src/github.com/getlantern/flashlight/ui/ui.go

```diff
@@ -0,0 +1,72 @@
+package ui
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/tarfs"
+	"github.com/skratchdot/open-golang/open"
+)
+
+const ()
+
+var (
+	log = golog.LoggerFor("ui")
+
+	localResourcesPath = filepath.Join(os.Getenv("GOPATH"), "src/github.com/getlantern/ui/app")
```

Comment:
Ahh, cool.  Sorry about that - restored now.  I think I missed it because I ditched the branch that bypassed tarfs.